docker/cli
# Fix always listing nodes in docker stack ps command

**- What I did**
* Fixed the `docker stack ps` command on Kubernetes which always failed with a user without node listing privilege.
* Fixed double error message output on `docker stack rm` command.

**- How to verify it**
1. Create a user on Kubernetes with only read access on a namespace
1. Deploy a stack on this namespace
1. With this user's credentials, execute a `docker stack ps` on this namespace -> it should not fail and list all the containers

**- Description for the changelog**
* Fix always listing nodes during docker stack ps command on Kubernetes.
* Fix outputting twice the docker stack rm error message 

**- A picture of a cute animal (not mandatory but encouraged)**

![image](https://user-images.githubusercontent.com/31478878/40722847-bc70d0be-641d-11e8-9216-608702857d33.png)
